docs/concepts/source-ip/source-ip.md

```diff
@@ -0,0 +1,300 @@
+---
+---
+
+{% capture overview %}
+Applications running in a Kubernetes cluster find and communicate with each
+other, and the outside world, through the Service abstraction. This document
+explains what happens to the source IP of packets sent to different types
+of Services, and how you can toggle this behavior according to your needs.
+{% endcapture %}
+
+{% capture body %}
+
+### Terminology
+
+Throughout this document, we will use the following terms:
```

Comment:
Avoid "we" constructs.

"This document makes use of the following terms:" is better.